src/components/pets/pets.css

```diff
@@ -56,3 +56,48 @@
     5px 5px 15px 5px rgba(0, 0, 0, 0);
   box-shadow: 0px 10px 13px -7px #000000, 5px 5px 15px 5px rgba(0, 0, 0, 0);
 }
+
+.image {
+  position: relative;
+  width: 400px;
```

Comment:
the width is kinda overlaps on mobile devices I think we can use 100%

Reply:
Width changed to 100%